### ISSUE TITLE
Timestamp not working on css assets

### ISSUE THREAD

**Issue by niclarke** [Compass/compass#75]
While setting up a new app I noticed that the timestamps were not appearing on the images (image_url in sass).

All of the urls were: http://domain/images/myimage.jpg?
and not http://domain/images/myimage.jpg?45372

Below is the fix:
  lib/compass/sass_extensions/functions/urls.rb
  line 119: File.mtime(real_path).strftime("%s")

  Changed to: File.stat(real_path).mtime.to_i

The error is that %s (small s) is not a valid format. There is a %S but I preferred the to_i route above as this matched how rails looked.

Nick

**Comment by chriseppstein** [comment]
%s totally works on my machine:

```
irb(main):001:0> real_path = "README.markdown"
=> "README.markdown"
irb(main):002:0> File.mtime(real_path).strftime("%s")
=> "1260897422"
```

But it's not mentioned in the official documentation -- which is strange because I'm pretty sure that's where I learned about it. Anyways, to_i is documented as working in the same way so I'll make this change.

**Comment by chriseppstein** [comment]
Cache buster timestamps not working in some cases. Closed by 05d89b082cbc947c5b7599dc06efb18f1e9f58c9.
